Source/Visit/VisitProposal.swift

```diff
@@ -4,10 +4,15 @@ public struct VisitProposal {
     public let url: URL
     public let options: VisitOptions
     public let properties: PathProperties
+    public let bundle: [String: Any]?
     
-    public init(url: URL, options: VisitOptions, properties: PathProperties = [:]) {
+    public init(url: URL, 
+                options: VisitOptions,
+                properties: PathProperties = [:],
+                bundle: [String: Any]? = nil) {
```

Comment:
> If bundle feels like odd language for the platform, feel free to rename.

It doesn't feel particularly odd. We could certainly rename it to `arguments` or `context` or something similar. While I like how it's consistent with Android naming, I don't feel too strongly about it.

> What actually is the bundle here?

So, for example, in HEY we have an import contacts flow that's something like this:

1. Present contacts from the user's phone (native: `CNContactPickerViewController`)
2. When user selects contacts and taps done, move to an upload progress screen (native: e.g. `UploadContactsViewcontroller`)

There's no easy path to move between screen 1 and 2 using `TurboNavigator` since screen 2 requires the contacts selected from screen 1.  So we'd need to tightly couple 1 and 2 (defeats the purpose of having the path config) or pass the contacts some other way (not super clean). 

Jay and I discussed this and Android has an easy way of doing this which is passing along the data required for setting up the screen alongside the route.

In the scenario above, we'd call something like `route(to: "/native/contacts/upload",  bundle: ["contacts" : selectedContacts])`. The bundle becomes a part of the `VisitProposal` so when the client needs to resolve the proposal in `handle(proposal: VisitProposal)` they have all the data needed to setup the new client.

Reply:
Ah! Sure, totally missed that. I'll rename to context. 👍🏻

Reply:
I don't think we should use `context`, since that conflicts with the existing first-class `context: "default"` and `context: "modal"` setup in the path config. Let's find something else that makes it clear how its intended to be used. I'm also find renaming the Android one in the future if we find a good name.

Reply:
Good point too! I'll try to find a good name and report back.

Reply:
I like this idea!
Regarding naming, what about `parameters` or `arguments`?

Reply:
`parameters` ended up being the winner. Thanks everyone!